lib/git-whistles/youtrack/api.rb

```diff
@@ -0,0 +1,80 @@
+require 'youtrack'
+require_relative 'ticket'
+require 'term/ansicolor'
+
+module Git::Whistles
+  module Youtrack
+    class Api
+      def get_ticket(id)
```

Comment:
please call this `find_ticket` or `find` or even `Ticket.find` as it will go more aligned with the current codebase